.gitignore

```diff
@@ -138,3 +138,6 @@ credentials.csv
 *.pub
 *.rdp
 *_rsa
+
+# IDE settings
+.idea
```

Comment:
I usually recommend folks put this kind of thing in a [global gitignore file](https://stackoverflow.com/questions/7335420/global-git-ignore).